src/fixes/leading_character.rs

```diff
@@ -0,0 +1,252 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct LeadingCharacterFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for LeadingCharacterFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "LeadingCharacter",
+        }
+    }
+}
+
+impl Fix for LeadingCharacterFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_line(&self, line: &mut LineEntry) -> Option<()> {
+        line.raw_string = line.raw_string.to_string();
```

Comment:
I don't fully understand why you do it? Can you explain me?

Reply:
That line was unnecessary, good catch, thank you :), I removed here: https://github.com/dotenv-linter/dotenv-linter/pull/259/commits/f683b815ba876f388ce3c1262d7a16b78a3aefc3 and I also extracted this function to remove the first character of the string, just in case it was unneccesarily confusing: https://github.com/dotenv-linter/dotenv-linter/pull/259/commits/3c16dc06639e0b2bb7c68c1ddd701c4dabde3bc0